superset-frontend/packages/superset-ui-demo/package.json

```diff
@@ -24,11 +11,24 @@
     "analysis",
     "data"
   ],
-  "license": "Apache-2.0",
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
```

Comment:
bycatch: since that repo is archived, I wonder if we should just point these to the main repo?